start-debugger.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Script to help start the users application along with
+# the .NET Core Stackdriver Debugger.
+#
+# Input should be the command to start the users donet
+# application as they normally would.  For example if
+# they would call 'dotnet HelloWorld.dll' they would
+# just run ''./start-debugger.sh dotnet HelloWorld.dll'
+
+start_command=
+while (( "$#" )); do
+  start_command="$start_command $1"
+  shift
+done
+
+# Start the users application to avoid any slow down 
+# while starting the debugger.
+$start_command &
+
+# Start the debugger and attach to the users process.
+APP_PID=$(echo $!)
```

Comment:
In Java, which is GA for Stackdriver Debugger, they recommend just launching the agent and providing a path of the app you want to attach to, so the agent will launch the app and then attach to it. I think we should do the same - it's both consistent with other languages and also make it naturally easier for the agent to track the exit code - in case customer's process is failing to start.

Reply:
We support this use case but do not have it as the default for .NET Core as the start up time is too long.  With this approach you may miss start up code but the start up time meets the start up requirements.